apcd_cms/src/client/src/hooks/entities/useEntities.ts

```diff
@@ -2,17 +2,21 @@ import { useQuery, UseQueryResult } from 'react-query';
 import { fetchUtil } from 'utils/fetchUtil';
 import { SubmitterEntityData, SubmitterDataPeriods } from '.';
 
-const getEntities = async () => {
+const getEntities = async (params: any) => {
   const url = `common_api/entities/`;
   const response = await fetchUtil({
     url,
+    params
   });
   return response.response;
 };
 
-export const useEntities = (): UseQueryResult<SubmitterEntityData> => {
+export const useEntities = (
+  from_admin: boolean | undefined
```

Comment:
Without this existing uses are failing to compile
```suggestion
  from_admin: boolean | undefined  = false
```